defaults/main.yml

```diff
@@ -20,3 +19,9 @@ mesos_zookeeper_masters: "zk://{{ zookeeper_hostnames }}/mesos"
 # Containerizer
 mesos_containerizers: "mesos"
 mesos_executor_timeout: "5mins"
+
+# Additional configurations
+mesos_configs: []
```

Comment:
Yeah, I just added empty list `[]` to make it clear that it's array.
Agree that `mesos_additional_configs` make more sense. I will update it.